src/ddb/ddb_cmd_options.c

```diff
@@ -10,14 +10,18 @@
 #include "ddb_cmd_options.h"
 #include "ddb_common.h"
 #define same(a, b) (strcmp((a), (b)) == 0)
+#define COMMAND_NAME_HELP "help"
 #define COMMAND_NAME_QUIT "quit"
 #define COMMAND_NAME_LS "ls"
 #define COMMAND_NAME_DUMP_SUPERBLOCK "dump_superblock"
-#define COMMAND_NAME_DUMP_ILOG "dump_ilog"
-#define COMMAND_NAME_DUMP_DTX "dump_dtx"
 #define COMMAND_NAME_DUMP_VALUE "dump_value"
 #define COMMAND_NAME_RM "rm"
 #define COMMAND_NAME_LOAD "load"
+#define COMMAND_NAME_DUMP_ILOG "dump_ilog"
+#define COMMAND_NAME_PROCESS_ILOG "process_ilog"
+#define COMMAND_NAME_RM_ILOG "rm_ilog"
+#define COMMAND_NAME_DUMP_DTX "dump_dtx"
+#define COMMAND_NAME_CLEAR_DTX "clear_dtx"
```

Comment:
Better to name it as "clear_committed_dtx" or "clear_cmt_dtx" since your patch does not support to clear active DTX entries.